src/content/docs/infrastructure/host-integrations/host-integrations-list/active-directory-integration.mdx

```diff
@@ -26,50 +23,71 @@ Built with our infrastructure agent, the Active Directory integration gives you
 
-* Follow our [guided install](https://one.newrelic.com/marketplace?state=8f14e646-461e-b010-4675-3a0658bb3d20) to instrument your system with the infrastructure agent. 
-* [Install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) via the command line using our docs.
+Run the following command in your CLI to clone the New Relic Active Directory integration repository:
 
-## Instrument Active Directory with New Relic [#instrument]
+```bash
+git clone https://github.com/newrelic/newrelic-active-directory-integration.git
+```        
 
-To instrument Active Directory with the infrastructure agent, follow the below procedures between this doc and the [New Relic Active Directory integration repository](https://github.com/newrelic/newrelic-active-directory-integration). 
+    </Step>
+    <Step>
 
-<CollapserGroup>
-    <Collapser
-    id="step1"
-    title="Step 1. Clone the repository"
-    >
-    Run the following command in your CLI to clone the New Relic Active Directory integration repository:
+## Copy configuration files [#copy-config]
+Copy the relevant configuration files from the repository to the agent, replacing `$ROOT_PATH` with the path where you cloned the repository:
 
-    ```bash
-    git clone https://github.com/newrelic/newrelic-active-directory-integration.git
-    ```
-    </Collapser>
+```powershell
+# integrations.d
+Copy-Item -Path "$ROOT_PATH\newrelic-active-directory-integration\integrations.d\*"     -Destination "C:\Program Files\New Relic\newrelic-infra\integrations.d" -Recurse
 
-    <Collapser
-    id="step2"
-    title="Step 2. Copy the configuration files to your host"
-    >
-    Copy the relevant configuration files from the repository to the agent, replacing `$ROOT_PATH` with the path where you cloned the repository:
+# logging.d
+Copy-Item -Path "$ROOT_PATH\newrelic-active-directory-integration\logging.d\*"     -Destination "C:\Program Files\New Relic\newrelic-infra\logging.d" -Recurse
+```        
 
-    ```powershell
-    # integrations.d
-    Copy-Item -Path "$ROOT_PATH\newrelic-active-directory-integration\integrations.d\*"     -Destination "C:\Program Files\New Relic\newrelic-infra\integrations.d" -Recurse
-    
-    # logging.d
-    Copy-Item -Path "$ROOT_PATH\newrelic-active-directory-integration\logging.d\*"     -Destination "C:\Program Files\New Relic\newrelic-infra\logging.d" -Recurse
-    ```
-    </Collapser>
-</CollapserGroup>
+    </Step>
+    <Step>
+## Install the integration [#install-io]
 
-Once you've followed all these steps, you can install the [Active Directory quickstart](https://newrelic.com/instant-observability/active-directory) from the New Relic I/O catalog to add the relevant dashboard and alerts.
+Install the [Active Directory quickstart](https://newrelic.com/instant-observability/active-directory) from the New Relic I/O catalog to add the relevant dashboard and alerts.         
```

Comment:
Two thoughts:

- Is the quickstart inextricable to the integration itself? i.e., you have incomplete data w/o the quickstart
- If it isn't, do we need to keep it in the main flow? I think if it isn't necessary for complete data, it can go into "Find/use data" as an _option_ for dashboarding.

My reason for asking is that there's a lot of granularity re: configuration, and the repo itself doesn't reference the IO page. You have more context, so it could be the repo is out of date, but if the repo is the source of truth that we're copying here, I'd err on the side of technical specificity btwn the configuration files over a dashboard quickstart tbh